introduction.md

```diff
@@ -1 +1,8 @@
 Hello, my name is David. My favorite Code School path so far is JavaScript, but I am really loooking forward to iOS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?